recipes/cellhint/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "cellhint" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f73d7c476732249383e69d0d977d7a574fd42ef2d126ea0e007913c09f993b1a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - leidenalg >=0.9.0
+    - numpy >=1.19.0
+    - openpyxl >=3.0.4
+    - pandas >=1.0.5
+    - python >=3.6
+    - scanpy >=1.7.0
+    - scikit-learn >=0.24.1
+    - python-annoy
```

The upstream project has a minimum version specified: https://github.com/Teichlab/cellhint/blob/c0795803e7da5a1ce864c09c7e3a70faae9e3818/requirements.txt#L7C1-L7C6

```suggestion
    - python-annoy >=1.17.0
```